### PR TITLE
fix: expand spec, keep server url

### PR DESCRIPTION
**Pull request by max107**
fix: keep server url in extended spec

### CODE REVIEW

File: openapi/parser/expand.go [hunk 1]
```diff
@@ -137,6 +137,7 @@ func (e *expander) Spec(api *openapi.API) (spec *ogen.Spec, err error) {
 
 func (e *expander) Server(s openapi.Server) (expanded ogen.Server, err error) {
 	expanded.Description = s.Description
+	expanded.URL = s.URL
```
Comment (tdakkota) [inline openapi/parser/expand.go:140]: Variable `template` already contains stringified server URL template.

This line should be at the end of the function:
```go
expanded.URL = template.String()
```

**Review by tdakkota — CHANGES_REQUESTED** [review]

File: openapi/server.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package openapi
 // Server represents parsed OpenAPI Server Object.
 type Server struct {
 	Name        string // optional,extension
+	URL         string
```
Comment (tdakkota) [inline openapi/server.go:6]: This field is unused.

**Review by tdakkota — CHANGES_REQUESTED** [review]
Could you rebase your PR?

Also, fix commit title according to [Conventional Commits](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum), please.

**Comment by max107** [comment]
https://github.com/ogen-go/ogen/pull/1079

**Pull request closed without merging**